qa/L0_client/test.sh

```diff
@@ -50,6 +50,101 @@ for l in libgrpcclient.so libgrpcclient_static.a libhttpclient.so libhttpclient_
     fi
 done
 
+# Test a simple app using Triton gRPC API
+client_lib=$(pwd)/triton_client/lib
+
+# Test linking against the shared library
+rm simple_grpc_client
+g++ grpc_client.cc -o simple_grpc_client -Itriton_client/include \
```

Comment:
-o grpc_test
The simple_ prefix is something we use for our examples so is confusing here.